### PR TITLE
[FUCK] [NO GBP] Fix Ghost Pods Speaking Over Common and Showing Ghost Roles on Manifest!

### PR DESCRIPTION
**Pull request by RimiNosha**
## About The Pull Request
WHYDIDITBREAKLIKETHISITWORKEDFINEONMYPCASIDEFROMTHEMANIFESTTHINGAAAAAAA

Anyways, it's fixed now!

## How This Contributes To The Skyrat Roleplay Experience
Bugfix.

## Changelog

:cl:
fix: Fixed ghost cryopods and cryo consoles, I swear I know how to code.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/cryosleep/code/cryopod.dm [hunk 1]
```diff
@@ -304,8 +306,14 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
...
 		crew_member["job"] = "N/A"
 
-	// Delete them from datacore.
+	// Delete them from datacore and ghost records.
 	var/announce_rank = null
+
+	for(var/datum/data/record/record as anything in GLOB.ghost_records)
+		if(record.fields["name"] == mob_occupant.real_name)
+			announce_rank = record.fields["rank"]
+			qdel(record)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/cryosleep/code/cryopod.dm]: This way you won't have nulls in that list. (I think that's how the remove proc is written).
```suggestion
			announce_rank = record.fields["rank"]
			GLOB.ghost_records.Remove(record)
			qdel(record)
```

File: modular_skyrat/modules/mapping/code/mob_spawns.dm [hunk 2]
```diff
@@ -501,10 +501,10 @@
 //CRYO CONSOLES
 /obj/machinery/computer/cryopod/interdyne
 	radio = /obj/item/radio/headset/interdyne
-	announcement_channel = RADIO_CHANNEL_INTERDYNE
+	announcement_channel = "Interdyne"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/mob_spawns.dm]: Why can't those be defines? If the content of the define wasn't correct, just make a new one with the proper name in it.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/mapping/code/mob_spawns.dm (hunk 2 above)
Comment (RimiNosha) [reply modular_skyrat/modules/mapping/code/mob_spawns.dm]: Announcement channel was set to null on live, and this is the only reason why I can think of, and it works on my machine<sup>:tm:</sup>, so yea...

Comment (Zonespace27) [reply modular_skyrat/modules/mapping/code/mob_spawns.dm]: on live it is now not null, instead it's not announcing it at all

**Pull request merged**